app/templates/request_purpose.hbs

```diff
@@ -34,7 +34,9 @@
             <section
               class="main-section request-purpose medium-10 medium-offset-1 large-10 large-offset-1 small-10 small-offset-1 columns">
               {{#unless cart.isEmpty}}
-              {{small-cart-toggle-view}}
+               {{#unless bookAppointment}}
```

Comment:
@akshaygwd use the `order_summary` partial here as well.

Reply:
`bookAppointment` where's this property defined @akshaygwd ?

Reply:
isAppointment property is not accessible in request_purpose.hbs file that why using bookAppointment query param for request_purpose.hbs file

Reply:
still you can use the partial.

Reply:
@akshaygwd use this in the partial `{{#unless is-or(isAppointment bookAppointment) }}`

so it will check for both properties. 

and you can use this partial in request-purpose as well.

cc- @abulsayyad123 @namrataukirde

Reply:
okay got it

Reply:
Never mind, now that the PR is merged, lets keep it as is and avoid the tight coupling that above suggestion might cause in the partial file.

Reply:
sure @abhinav098